Makefile

```diff
@@ -137,4 +133,49 @@ PACKAGES_UNIT=$(shell go list ./...)
 
 test: test-unit
 test-unit:
-	@VERSION=$(VERSION) go test -mod=readonly $(PACKAGES_UNIT)
\ No newline at end of file
+	@VERSION=$(VERSION) go test -mod=readonly $(PACKAGES_UNIT)
+
+###############################################################################
+###                                Localnet                                 ###
+###############################################################################
+
+# Build image for a local testnet
+localnet-build:
+	docker build -f Dockerfile -t eve-node .
+
+# Start a 4-node testnet locally
+localnet-start: localnet-clean
```

Comment:
docker-compose down
sudo rm -rf build
rm mnemonics.txt
rm: cannot remove 'mnemonics.txt': No such file or directory
make: *** [Makefile:160: localnet-clean] Error 1

Should run the add-keys.sh script here automatically? 
when I run manually

`Could not open file build/node3/eved/key_seed.json: No such file or directory`

also mainnet = eve-1, so should do eve-t1 or l1 for test/local version

Reply:
should not have sudo in a makefile